docker/Dockerfile

```diff
@@ -6,22 +6,22 @@ FROM debian:buster
 # Install fuse-overlayfs and Tern dependencies
 RUN apt-get update && \
     apt-get -y install \
-    attr \
-    findutils \
-    git \
-    gnupg2 \
-    python3 \
-    python3-pip \
-    python3-setuptools \
-    tar \
-    util-linux \
+      attr \
+      findutils \
+      git \
+      gnupg2 \
+      python3 \
+      python3-pip \
+      python3-setuptools \
+      tar \
+      util-linux \
```

Comment:
Looks like a tab was accidentally added?

Reply:
Normally I would always do some extra indenting after the`apt-get -y install \` line, so it's clear that the lines below belong to the `apt-get` command. This is a matter of taste, I can also remove it.